metpy/calc/basic.py

```diff
@@ -220,3 +222,28 @@ def heat_index(temperature, rh, mask_undefined=True):
             hi = units.Quantity(masked_array(hi, mask=mask), hi.units)
 
     return hi
+
+
+@exporter.export
+def pressure_to_height_std(pressure):
+    '''Convert pressure data to heights using the U.S. standard atmosphere.
+
+    Parameters
+    ----------
+    pressure: array_like
```

Comment:
It's entirely possible sphinx is mad about the lack of a space between `pressure` and the `:`.